### DISCUSSION TITLE
(Solved) Gnome theme

### DISCUSSION THREAD

**Discussion by straffer44** [SpiralLinux/SpiralLinux-project#391] [General]
I am trying to use Adwaita as the theme and have set all in tweaks (shell set to default as legacy set to Adwaita).  I have even removed Yaru themes (shell and gtk) using synaptic.  For some reason red highlights persist in files, text editor, extensions and extensions manager.  Everything else is using blue highlights.
What am i missing?
First time using Gnome I have sprial xfce installed on another computer. Thanks for all the hard work creating Sprial.

**Comment by geckolinux** [comment]
Hi there, to remove it you can delete or edit `~/.config/gtk-4.0/gtk.css` .

**Comment by straffer44** [comment]
Thanks so much, removing the file worked ;)
Theme changes in gnome seem more complicated than xfce.

**Reply by geckolinux** [reply]
It's specifically a GTK4 thing, which Gnome uses more heavily than XFCE.
